### ISSUE TITLE
[Dropzone] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1275]
### Feedback

- The dropzone takes focus - then each of the other subsequent links take focus. Unless you allow copy/paste to work (unsure if it does) there is no need for the dropzone to take focus
- tabindex=0 should not be applied to something without a role. If this is an interactive element then it needs and interactive role

### Path forward

1. Docs apply `tabindex` from the outside, should be removed in this context.
2. Usage of dropzone without focusable child content should be documented in some way.

cc @jnurthen

**Comment by Westbrook** [comment]
Related types for dispatched events are wrong... https://github.com/adobe/spectrum-web-components/blob/9ab5f05eacbe11e30cbf7158b032733c781f1104/packages/dropzone/src/Dropzone.ts#L156-L163
